src/components/DropDownMenu.svelte

```diff
@@ -12,7 +12,15 @@
 
 <div>
   <label class="label block mb-3.5" for={id}>{label}</label>
-  <select {id} bind:value={selected} {required} {disabled} on:change={onChange} on:select={onSelect}>
+  <select
+    {id}
+    bind:value={selected}
+    {required}
+    {disabled}
+    on:change={onChange}
+    on:select={onSelect}
+    class="cursor-pointer"
```

Comment:
Added this to override what I had to do to make the modal pass the accessibility errors.